.changeset/eleven-mirrors-guess.md

```diff
@@ -0,0 +1,5 @@
+---
+"saleor-dashboard": minor
```

Comment:
So I assume that you don't want to port this pr to older releases eg. 3.18, 3.19?

Reply:
I will cherry pick it later

Reply:
But it's minor, so you need to change it to patch then